node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/task/util/file/UnixPath.java

```diff
@@ -171,6 +189,12 @@ public boolean deleteIfExists() {
         }
     }
 
+    /** This path must be on the same file system as the to-path. */
+    public UnixPath atomicMove(Path to) {
+        uncheck(() -> Files.move(path, to, StandardCopyOption.ATOMIC_MOVE));
+        return this;
```

Comment:
I would expect this to return the `new UnixPath(to)`?

Reply:
Agree